src/main/java/com/google/maps/model/OpeningHours.java

```diff
@@ -0,0 +1,58 @@
+package com.google.maps.model;
+
+public class OpeningHours {
+  /**
+   * openNow is a boolean value indicating if the place is open at the current time.
+   */
+  public Boolean openNow;
+
+  static public class Period {
+    static public class OpenClose {
+      public enum DayOfWeek {
```

Comment:
I don't really have a strong opinion, but would it be worth just using `java.time.DayOfWeek`? It has slightly different numbered semantics.